tooling/typescript/internal-package.json

```diff
@@ -5,6 +5,7 @@
     /** Emit types for internal packages to speed up editor performance. */
     "declaration": true,
     "declarationMap": true,
+    "emitDeclarationOnly": true,
     "noEmit": false,
```

Comment:
What happens if we remove `"declaration": true`? I know `"noEmit": false` is there to override the extended base config so it should probably stay.

Reply:
cause we need declaration files